### ISSUE TITLE
Fix mayor endpoint to work like the service

### ISSUE THREAD

**Issue by matthias-luger** [Coflnet/SkyApi#3]
The mayor endpoint was slightly changed. While the path and the way the params are written is fine for me, the datatype was changed to int32 which is too small for timestamps.

![Image](https://user-images.githubusercontent.com/58751503/191315442-9f68733f-8f7a-413a-9bfb-cbd8addfac7a.png)

![Image](https://user-images.githubusercontent.com/58751503/191315460-44c9b3a1-22b9-4b51-9870-d4471fe9a5ee.png)

**Comment by matthias-luger** [comment]
@Ekwav 
https://github.com/Coflnet/sky-mayor/pull/7 added the format property to the swagger it should now generate it correctly.
Please notify me when the new version is live

**Comment by Ekwav** [comment]
@matthias-luger the swagger is invalid. the `generate.sh` outputs the following
```
Exception in thread "main" org.openapitools.codegen.SpecValidationException: There were issues with the specification. The option can be disabled via validateSpec (Maven/Gradle) or --skip-validate-spec (CLI).
 | Error count: 2, Warning count: 0
Errors: 
        -attribute paths.'/electionPeriod/range/{from}/{to}/'. Declared path parameter from needs to be defined as a path parameter in path or operation level
        -attribute paths.'/electionPeriod/range/{from}/{to}/'. Declared path parameter to needs to be defined as a path parameter in path or operation level
```

**Comment by matthias-luger** [comment]
@Ekwav I think I found the bug and pushed a fix in main. Please try again

**Comment by Ekwav** [comment]
Worked, released client bindings v0.1.0 and updated https://github.com/Coflnet/SkyApi/commit/01d38fc6f817413133fcce9b1a083090fcc8063a
